sql/updates/world/3.3.5/xxxx_xx_xx_xx.sql

```diff
@@ -0,0 +1,7 @@
+DELETE FROM `trinity_string` WHERE `entry` = 5089;
+INSERT INTO `trinity_string` (`entry`,`content_default`) VALUES
+(5089, 'Template StringID: %.*s\r\nSpawn StringID: %.*s\r\nScript StringID: %.*s');
+
+ALTER TABLE `creature` ADD `StringId` varchar(64) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL AFTER ScriptName;
```

Comment:
`ScriptName` backticks if needed

Reply:
You guys give me enough information to say "What?"

Reply:
Thats what I get for copying someone elses SQL.